mage_ai/cache/block_action_object/__init__.py

```diff
@@ -254,48 +254,46 @@ async def initialize_cache_for_all_objects(self) -> None:
                 include_content=True,
             )
 
-        for path_dict in build_repo_path_for_all_projects(mage_projects_only=True).values():
-            repo_path = path_dict['full_path']
-
-            for block_type in BlockType:
-                file_directory_name = Block.file_directory_name(block_type)
-                directory_full_path = os.path.join(repo_path, file_directory_name)
-
-                for path in Path(directory_full_path).rglob('*'):
-                    if not path.is_file():
-                        continue
-
-                    block_file_absolute_path = path.absolute()
-                    d = parse_block_file_absolute_path(block_file_absolute_path)
-
-                    file_path = d.get('file_path')
-                    filename = d.get('filename')
-                    filename_parts = d.get('filename_parts')
-                    language = d.get('language')
-
-                    if '__init__.py' == filename:
-                        continue
-
-                    if not language:
-                        continue
-
-                    key = self.build_key_for_block_file(
-                        file_path,
-                        block_type,
-                        language,
-                        filename,
-                    )
-
-                    content = None
-                    with open(block_file_absolute_path, 'r') as f:
-                        content = f.read()
-
-                    mapping[OBJECT_TYPE_BLOCK_FILE][key] = dict(
-                        content=content,
-                        file_path=file_path,
-                        language=language,
-                        type=block_type,
-                        uuid='.'.join(filename_parts[:-1]),
-                    )
+        repo_path = base_repo_path()
```

Comment:
This line was the main change. `build_repo_path_for_all_projects(mage_projects_only=True).values()` wasn't returning anything and was also causing the same directories to be iterated through multiple times.